### DISCUSSION TITLE
Membres Agile Toulouse

### DISCUSSION THREAD

**Discussion by guillaumeagile** [AgileToulouse/editionJuin2023#4] [General]
Merci de vous identifier ici avec votre login Github pour que je vous ajoute à la team Orga

**Comment by ClaireMalha** [comment]
Nouvelle sur Github, Tu peux m'ajouter dans la team Orga ?

**Reply by guillaumeagile** [reply]
fait, il faut que tu confirmes ta participation

**Reply by s31db** [reply]
Coucou Guillaume
Sam

**Reply by guillaumeagile** [reply]
Salut Sam, tu as recu une invitation ici dans Github a faire partie de l'équipe, il faut que tu la valides ;)
